src/pages/validator/external-chains/blast.mdx

```diff
@@ -0,0 +1,185 @@
+
+- [Setup your Axelar validator](/validator/setup)
+- Minimum hardware requirements: 8 AWS vCPU+, 8GB RAM, 512GB of SSD free storage space.
+- MacOS or Ubuntu 22.04 (tested on 22.04)
+- [Official Github Documentation](https://github.com/blast-io/deployment)
+- [Official Documentation for Node Providers](https://docs.blast.io/tools/node-providers)
+
+## Prerequisites
+
+- [Common Docker Compose Commands](./optimism#common-node-operations)
+```bash
+sudo apt-get install jq -y
+snap install docker
+```
+
+## Clone Repo
+
+```bash
+git clone https://github.com/blast-io/deployment.git blast-node
+cd blast-node
+```
+
+## Set Node Config Environment Variables
+```bash
+cp .env.example .env
+```
+
+Set the following config options in `blast-node/.env.testnet` or `blast-node/.env`:
+```bash
+# NETWORK should be mainnet or sepolia
+NETWORK=mainnet
+# Relative path to the directory that will store chain data
+GETH_DATA_DIR=blast-geth-data
+# Your Sepolia L1 RPC endpoint
+L1_RPC_URL="your_l1_rpc_url"
+# The type of RPC provider, valid options are alchemy, quicknode, infura, parity, nethermind, debug_geth, erigon, basic, any
+# Axelar Team recommends you to run your own
+L1_RPC_KIND="any"
+```
+
+## Start the node
+```bash
+docker compose up -d
+```
+
+This should show an output like this:
+```bash
+[+] Running 6/6
+ ✔ Network blast-node_blast-net         Created                                                                                                              0.1s
+ ✔ Network blast-node_default           Created                                                                                                              0.1s
+ ✔ Container blast-node-generate-jwt-1  Exited                                                                                                               1.0s
+ ✔ Container blast-node-geth-init-1     Exited                                                                                                               1.8s
+ ✔ Container blast-node-blast-geth-1    Started                                                                                                              2.0s
+ ✔ Container blast-node-op-node-1       Started                                                                                                              2.3s                                                                                                      11.2s
+```
+
+After it is done, verify by listing the services and their status
+ ```bash
+docker compose ps
+```
+
+You should see these 2 services running
+```bash
+NAME                      IMAGE                                    COMMAND                  SERVICE             CREATED              STATUS              PORTS
+blast-node-blast-geth-1   blastio/blast-geth:testnet-sepolia       "geth --datadir=/bla…"   blast-geth          About a minute ago   Up 58 seconds       8545-8546/tcp, 30303/tcp, 0.0.0.0:9545-9546->9545-9546/tcp, :::9545-9546->9545-9546/tcp, 30303/udp
```

Comment:
As per latest docker compose file from Blast docs https://github.com/blast-io/deployment/blob/master/docker-compose.yml#L25 the Image name would be  blastio/blast-geth:mainnet or  blastio/blast-geth:sepolia and not ` blastio/blast-geth:testnet-sepolia`

Reply:
Yeah, let me update that, it changed after the their commit yesterday